package.json

```diff
@@ -35,7 +35,7 @@
     "imagesloaded": "^4.1.4",
     "immer": "^9.0.6",
     "jquery-bridget": "2.0.1",
-    "jquery-modal": "^0.9.2",
+    "jquery-modal": "0.5.9",
```

Comment:
What does the caret mean? Just curious.

Reply:
I was thinking that the caret would allow packages with minor version changes so I dropped it.  But after checking to be sure, it appears that it is more conservative on "0.x.x" versions.  So I guess we could've left it in. Though in our case it wouldn't matter since there's no 0.5.10, etc.:

https://medium.com/@Hardy2151/caret-and-tilde-in-package-json-57f1cbbe347b

Reply:
Sounds good.

Reply:
Looks like I lied to you. There are versions 0.5.10 and 0.5.11. I was going from memory and thought we were at the latest in the 0.5.x series.